### ISSUE TITLE
Java: Intermittent IllegalEntityException when creating an epoch receiver

### ISSUE THREAD

**Issue by gordontyler** [Azure/azure-event-hubs#177]
Sometimes, when starting my application that creates an epoch receiver, it will fail with the following exception:

```
java.util.concurrent.CompletionException: com.microsoft.azure.servicebus.IllegalEntityException: The session channel '65535' cannot be found in connection 'in-connection15019467(trinity-data-ingest)'., errorContext[NS: trinity-data-ingest.servicebus.windows.net, PATH: browser-events/ConsumerGroups/$Default/Partitions/1, REFERENCE_ID: 02b4f4_06ba16a77af741f094c5047fb0706467_G44, R_TYPE: EPOCH]
        at java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:292)
        at java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:308)
        at java.util.concurrent.CompletableFuture.uniAccept(CompletableFuture.java:647)
        at java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:632)
        at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:474)
        at java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:1977)
        at com.microsoft.azure.servicebus.ExceptionUtil.completeExceptionally(ExceptionUtil.java:104)
        at com.microsoft.azure.servicebus.MessageReceiver.onOpenComplete(MessageReceiver.java:320)
        at com.microsoft.azure.servicebus.MessageReceiver.onError(MessageReceiver.java:383)
        at com.microsoft.azure.servicebus.MessageReceiver.onError(MessageReceiver.java:349)
        at com.microsoft.azure.servicebus.MessageReceiver.onClose(MessageReceiver.java:650)
        at com.microsoft.azure.servicebus.amqp.BaseLinkHandler.processOnClose(BaseLinkHandler.java:56)
        at com.microsoft.azure.servicebus.MessagingFactory.onConnectionError(MessagingFactory.java:234)
        at com.microsoft.azure.servicebus.amqp.ConnectionHandler.onConnectionRemoteClose(ConnectionHandler.java:117)
        at org.apache.qpid.proton.engine.BaseHandler.handle(BaseHandler.java:128)
        at org.apache.qpid.proton.engine.impl.EventImpl.dispatch(EventImpl.java:108)
        at org.apache.qpid.proton.reactor.impl.ReactorImpl.dispatch(ReactorImpl.java:304)
        at org.apache.qpid.proton.reactor.impl.ReactorImpl.process(ReactorImpl.java:272)
        at com.microsoft.azure.servicebus.MessagingFactory$RunReactor.run(MessagingFactory.java:341)
        at java.lang.Thread.run(Thread.java:745)
Caused by: com.microsoft.azure.servicebus.IllegalEntityException: The session channel '65535' cannot be found in connection 'in-connection15019467(trinity-data-ingest)'., errorContext[NS: trinity-data-ingest.servicebus.windows.net, PATH: browser-events/ConsumerGroups/$Default/Partitions/1, REFERENCE_ID: 02b4f4_06ba16a77af741f094c5047fb0706467_G44, R_TYPE: EPOCH]
        at com.microsoft.azure.servicebus.ExceptionUtil.toException(ExceptionUtil.java:38)
        at com.microsoft.azure.servicebus.MessageReceiver.onError(MessageReceiver.java:348)
        ... 10 more
```

if I restart the app a few times, it will eventually succeed.

**Comment by SreeramGarlapati** [comment]
We fixed this in 0.7.7 - can you please run your reproducer code and validate !?

**Comment by gordontyler** [comment]
I am unable to reproduce this error with 0.7.7.
